setup.cfg

```diff
@@ -87,7 +87,7 @@ ml = dask-ml>=2022.5.27
 perf = rtree
     pygeos
     netCDF4
-    ray
+    ray>=2.3.0
```

Comment:
The load tests failed with a lower version of `ray` and the latest version of `dask`.